training/src/main/java/wolox/training/repositories/UserRepository.java

```diff
@@ -0,0 +1,13 @@
+package wolox.training.repositories;
+
+import java.util.Optional;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Component;
+import wolox.training.models.User;
+
+@Component
+public interface UserRepository extends CrudRepository {
```

Comment:
I think you need to add the types in CrudRepository (CrudRepository<User, Integer>).